core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5201,7 +5202,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String WORKER_KEYTAB_FILE = "alluxio.worker.keytab.file";
     public static final String WORKER_MASTER_CONNECT_RETRY_TIMEOUT =
         "alluxio.worker.master.connect.retry.timeout";
-    public static final String WORKER_MEMORY_SIZE = "alluxio.worker.memory.size";
+    public static final String WORKER_MEM_SIZE = "alluxio.worker.memory.size";
```

Comment:
nit: WORKER_MEM_SIZE -> WORKER_MEMORY_SIZE to keep consistent with our naming convention